.github/workflows/ci.yml

```diff
@@ -41,8 +43,25 @@ jobs:
     strategy:
       matrix:
         include:
-          - elixir: 1.10.3
+          - elixir: 1.11.2
             otp: 23.0.2
     steps:
       - uses: actions/checkout@v2.3.1
       - run: mix format --check-formatted
+
+  compile_with_warnings_as_errors:
```

Comment:
This shouldn't be a different build, should it? 🤔

Reply:
Meh... maybe not but I can refactor this whole file. Can we merge this and do a single pass?